### PR TITLE
chore: remove need for Spring context in unit test.

### PR DESCRIPTION
**Pull request by fhuaulme**
**Link to the JIRA issue**
https://jira.talendforge.org/browse/TDP-XXXX (no jira)

**Please check if the PR fulfills these requirements**
- [ ] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] The code coverage on new code is > 75 % for backend and > 95% for frontend
- [ ] The new code does not introduce new technical issues (sonar / eslint)
- [ ] Functional tests have been performed
- [ ] Docker configuration files for config-std or config-cloud profiles are impacted

**Please check the browsers you've tested on**
- [ ] Chrome, Firefox, Safari, Edge, IE11
- [ ] No, that's bad, this PR should not be merged !
- [ ] No, and no need to (backend changes only)

### CODE REVIEW

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/804/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/804/)

File: dataprep-backend-service/src/test/java/org/talend/dataprep/api/preparation/PreparationUtilsTest.java [hunk 1]
```diff
@@ -22,25 +22,44 @@
...
+import org.mockito.runners.MockitoJUnitRunner;
+import org.talend.dataprep.Mocks;
 import org.talend.dataprep.api.service.info.VersionService;
 import org.talend.dataprep.preparation.FixedIdPreparationContent;
 import org.talend.dataprep.preparation.FixedIdStep;
-import org.talend.dataprep.preparation.store.PersistentPreparationRepository;
 import org.talend.dataprep.preparation.store.PreparationRepository;
+import org.talend.dataprep.preparation.store.inmemory.InMemoryPreparationRepository;
 
-public class PreparationUtilsTest extends ServiceBaseTest {
```
Comment (aleboulanger) [inline dataprep-backend-service/src/test/java/org/talend/dataprep/api/preparation/PreparationUtilsTest.java]: @fhuaulme 
`org.talend.dataprep.api.preparation.PreparationUtilsTest#scatterPreparationSave` KO

File: dataprep-backend-service/src/test/java/org/talend/dataprep/cache/file/ResourceLoaderContentCacheTest.java [hunk 2]
```diff
@@ -28,25 +28,32 @@
...
+import org.talend.dataprep.cache.loader.ResourceLoaderContentCache;
 
 /**
  * Standard tests for a {@link ContentCache}
  *
  * @see LocalContentCacheTest
  */
-public abstract class ContentCacheTests extends ServiceBaseTest {
+@RunWith(MockitoJUnitRunner.class)
+public class ResourceLoaderContentCacheTest {
```
Comment (aleboulanger) [inline dataprep-backend-service/src/test/java/org/talend/dataprep/cache/file/ResourceLoaderContentCacheTest.java]: Test is KO

File: dataprep-backend-service/src/test/java/org/talend/dataprep/cache/file/LocalContentCacheTest.java [hunk 3]
```diff
@@ -18,5 +18,5 @@
  * Implementation of tests for local content cache.
  */
 @TestPropertySource(properties = { "content-service.store=local", "content-service.store.local.path=${java.io.tmpdir}/dataprep" })
-public class LocalContentCacheTest extends ContentCacheTests {
+public class LocalContentCacheTest extends ResourceLoaderContentCacheTest {
```
Comment (aleboulanger) [inline dataprep-backend-service/src/test/java/org/talend/dataprep/cache/file/LocalContentCacheTest.java]: Test is ko

**Comment by fhuaulme** [comment]
Too old. Requires a new attempt.

**Pull request closed without merging**